ddtrace/contrib/botocore/services/bedrock.py

```diff
@@ -0,0 +1,293 @@
+import json
+import sys
+from typing import Any
+from typing import Dict
+from typing import List
+
+from ddtrace import Span
+from ddtrace.contrib._trace_utils_llm import BaseLLMIntegration
+from ddtrace.vendor import wrapt
+
+from ....internal.schema import schematize_service_name
+
+
+class _BedrockIntegration(BaseLLMIntegration):
+    _integration_name = "bedrock"
+
+
+class TracedBotocoreStreamingBody(wrapt.ObjectProxy):
+    #  Currently the corresponding span finishes only if
+    #  1) the user fully consumes the stream body
+    #  2) error during reading
+
+    def __init__(self, wrapped, span, integration):
```

Comment:
Note: need to look at https://github.com/DataDog/dd-trace-py/blob/main/ddtrace/tracing/trace_handlers.py#L78 as an example

Reply:
Tested locally, this is a non-issue as checking for `len()` or `streaming_body.__len__` results in an attribute error without having to override `__getattribute__`.